p751toolbox/field_decl.go

```diff
@@ -2,6 +2,13 @@
 
 package p751toolbox
 
+import (
+    "golang.org/x/sys/cpu"
+)
+
+var hasADX = cpu.X86.HasADX
```

Comment:
Copying value of one variable into another doesn't seem to be needed

Reply:
I'd prefer to use `cpu.X86.HasSomething` directly in the assembly, but the Go assembler doesn't seem to allow a `.` in the variable name. I could not find if there's a syntax with `·` that allows this.

Reply:
I know. What I mean - there is no need for hasADX to exist